libraries/AP_BattMonitor/AP_BattMonitor_ACS37800.cpp

```diff
@@ -0,0 +1,187 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "AP_BattMonitor_ACS37800.h"
```

Comment:
This needs guards for when AP_BATTERY_ACS37800_ENABLED is not set - see AP_Battmonitor_Torqeedo.cpp to see how it's done.

Similarly for the header.